.travis.yml

```diff
@@ -14,26 +14,24 @@ env:
   - secure: "OeTwSjUtoQ567bboQKwf5rNAh2WJRuZu3SLB4+hhRj8S5FMHmqpDtezPa5Shex/a+6hT9DGfdag2ybxfLdR+tKphUb6VhuLsrETTz5DQFE480MrhtpaIxWMBVlemwIsA4eweL4weAp9xlapOPx+qs4oG12NBqLTNjOAW48NW4eD5E47j7/o1jA4YTZ59XxZXuqGjQTZopb5tqkaYmrEyhBk+xyzsVjWHSTeHlS5iuG9OBflRbJuzrmfDqLr3xdPivyLmFBFMjI08sCgpkeq1J4n6EJAuE4IMpWVYPims7lo2j2bPPH0dWfBYeoT4sCLItgqUISY97DlP0VjkbO8KwE/C0MUOFgrxdk58z1XRa5YKrJWcE5duw+6RQTq2FBjuWx0C2j4eGY049kyZuy3bcDJTsD4oGF1pRNa/CBiSBB6oOkjAh7X4evk4ve5m3jNGI865Y0YgTzNxgH0Cqm8gFTb6CIlJcLoKfg9PscmZvpPNahxX1TdN23Ku6mHIWHKRZcMo+mxcCAJCfQdZZ427ueEWRGtjBtPy8Q0ehc9CsicPNukMmJtKUObQIROxIWkrzUQBO9bLJNQACcCtUqRKp2wg0S5PMgPWidqkKNRSGbBOX5ew9RETsefaB3D2uLIgabhrJ3Lki0fTDDm+fTazVQhdTHlBqRZ5zfo+fFhQ3Ok="
   - secure: "VMteMjD5Z4iePR2oQRfZsBMX7QDtZiVpyd0btHomwAYht5OTVuSwQHRC+yvOBx+0HxL4MeOBBbo9fMFC72KadOJ9zMU9dnilwljBzRiHHNWrnubP35fp3rJtZaYS9CWdxIqSLNwKJXfiLVDfZn4UEWgP89vx5ZVizY0g2CS85i9/XjCOPu3KRntmBTCZGLH1pe+BIhc2W1M5HeNLWyFR1T1MmGehMg5Nt4lt53HMU4oFgdWQ1sGFkhhiYlKFO0KtLejVof2WARIuGRS10Qt/xBk5xLle8qCIyZRgRrLgcqWUk4wDKmCS3DiNj3rVXCRbLIlYFwJYUoJYIfY1NVvTrBOdneybyAls4IjbeKiz6TgrHKhplxx8Jt6+u6TMvvN83AqO53R801BP+IvSuMv7VqfRpqUV+S62a8rxoxZnQe+QGuzEL32fEG8J+mUMZBejKZTFGiw1vNwpPrNCOCheXqUZxqaERY35cFkesMCNoPTBAWUY8oY87CzQQMRIOnIIKAog9Uhi0hjNztxMk9/MfUtEsIRx6NKeBYBTc0jjcrXY/NDyGCB2E/S7I1jNqJk4uskHiYGr+tw/4q4rFDWSmBNGtdQtZ0mLpEb2ZuVaUtY+raH8PZOvba+/t7aaHfxFWuuGnqih8M7f58aRu/e75wfhikjXe1IvtQPSx8LGLzM="
   matrix:
-    - LATEST=0 SHOPWARE_VERSION=5.5.2 SHOPWARE_SHORT_VERSION=55 PHP_VERSION=7.3 PHP_SHORT_VERSION=73
-    - LATEST=1 SHOPWARE_VERSION=5.5.2 SHOPWARE_SHORT_VERSION=55 PHP_VERSION=7.2 PHP_SHORT_VERSION=72
-    - LATEST=0 SHOPWARE_VERSION=5.5.2 SHOPWARE_SHORT_VERSION=55 PHP_VERSION=7.1 PHP_SHORT_VERSION=71
-    - LATEST=0 SHOPWARE_VERSION=5.5.2 SHOPWARE_SHORT_VERSION=55 PHP_VERSION=7.0 PHP_SHORT_VERSION=70
-    - LATEST=0 SHOPWARE_VERSION=5.5.2 SHOPWARE_SHORT_VERSION=55 PHP_VERSION=5.6 PHP_SHORT_VERSION=56
+    - LATEST=1 SHOPWARE_VERSION=5.6.7 SHOPWARE_SHORT_VERSION=56 PHP_VERSION=7.3 PHP_SHORT_VERSION=73
+    - LATEST=0 SHOPWARE_VERSION=5.6.7 SHOPWARE_SHORT_VERSION=56 PHP_VERSION=7.2 PHP_SHORT_VERSION=72
+    - LATEST=0 SHOPWARE_VERSION=5.5.10 SHOPWARE_SHORT_VERSION=55 PHP_VERSION=7.3 PHP_SHORT_VERSION=73
+    - LATEST=0 SHOPWARE_VERSION=5.5.10 SHOPWARE_SHORT_VERSION=55 PHP_VERSION=7.2 PHP_SHORT_VERSION=72
+    - LATEST=0 SHOPWARE_VERSION=5.5.10 SHOPWARE_SHORT_VERSION=55 PHP_VERSION=7.1 PHP_SHORT_VERSION=71
+    - LATEST=0 SHOPWARE_VERSION=5.5.10 SHOPWARE_SHORT_VERSION=55 PHP_VERSION=7.0 PHP_SHORT_VERSION=70
```

Comment:
ist das nicht auch schon EOL?

Reply:
7.1 ist auch EOL. Haben wir da noch Projekte mit?

Reply:
Could be possible that there are projects which are in the upgrade phase at the moment.

But I removed 7.0